app/build.gradle

```diff
@@ -26,12 +26,14 @@ android {
 }
 
 dependencies {
-    implementation fileTree(dir: 'libs', include: ['PushIOManager.aar'])
-    implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation fileTree(dir: 'libs', include: ['PushIOManager-6.51.aar'])
+    implementation 'androidx.appcompat:appcompat:1.4.1'
     implementation 'com.android.support.constraint:constraint-layout:2.0.4'
-    implementation 'com.gitlab.bluedotio.android:point_sdk_android:15.3.1'
-    implementation 'com.google.firebase:firebase-core:18.0.2'
-    implementation 'com.google.firebase:firebase-messaging:21.0.1'
+    implementation 'com.gitlab.bluedotio.android:point_sdk_android:15.4.0'
+    implementation 'com.google.firebase:firebase-core:20.1.0'
+    implementation 'com.google.firebase:firebase-messaging:23.0.0'
+    implementation 'com.google.android.gms:play-services-base:18.0.1'
+    implementation 'androidx.core:core:1.7.0'
```

Comment:
What has changed that we need androidx core now?

Reply:
Its added based on the step-by-step guideline shared by [Responsys](https://docs.oracle.com/en/cloud/saas/marketing/responsys-develop-mobile/android/step-by-step.htm)

Reply:
Ahh, of course, the sdk is an aar, so they cannot package their own dependencies. 😞